atom/browser/api/atom_api_app.cc

```diff
@@ -699,6 +700,12 @@ void App::SetPath(mate::Arguments* args,
     args->ThrowError("Failed to set path");
 }
 
+base::FilePath App::GetInitialDownloadPath() {
```

Comment:
isn't this already accessible with `getPath('downloads')`?

Reply:
yeah, you are right!!! I'm going to close this PR and update PR of browser-laptop